hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/BlockingRpcConnection.java

```diff
@@ -667,46 +685,34 @@ private void readResponse() {
         // this connection.
         int readSoFar = getTotalSizeWhenWrittenDelimited(responseHeader);
         int whatIsLeftToRead = totalSize - readSoFar;
+        LOG.debug("Unknown callId: " + id + ", skipping over this response of " + whatIsLeftToRead
+          + " bytes");
         IOUtils.skipFully(in, whatIsLeftToRead);
         if (call != null) {
           call.callStats.setResponseSizeBytes(totalSize);
-          call.callStats
-            .setCallTimeMs(EnvironmentEdgeManager.currentTime() - call.callStats.getStartTime());
```

Comment:
was there a reason to remove this here?

Reply:
As said above, this is done in AbstractRpcClient, so we do not need to do it in RpcConnection imlementation.

https://github.com/apache/hbase/blob/3cfc92fa427441f1991df6ce5e6b95618710ea91/hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/AbstractRpcClient.java#L380